packages/backend/src/core/reports/AggregatedReportUpdater.ts

```diff
@@ -51,7 +53,10 @@ export class AggregatedReportUpdater {
   async update(timestamp: UnixTime) {
     this.logger.debug('Update started', { timestamp: timestamp.toNumber() })
 
-    const reports = await this.reportUpdater.getReportsWhenReady(timestamp)
+    const reports = [
```

Comment:
I am not sure whether it will work, we use Promise.all for such cases, see ReportUpdater.update